### ISSUE TITLE
[Enhancement] Center the article on the page

### ISSUE THREAD

**Issue by K4LCIFER** [Nutomic/ibis#50]
Currently, all elements are left-justified on the webpage. The article itself should be centered to the page with the text left justified within (assuming the language that it's written in is left-justified, ofc).

**Comment by Nutomic** [comment]
Tried this but it didnt look good. Instead I removed the max width entirely, seeing as wikipedia also uses the full width.

**Comment by K4LCIFER** [comment]
> Instead I removed the max width entirely

I don't think this is a good idea as really wide articles start to affect readability (not to mention they aren't the nicest looking).

> seeing as wikipedia also uses the full width.

It actually doesn't anymore:
![image](https://github.com/user-attachments/assets/12fee325-904c-4f6c-8ead-449b8bff71a6)
They do allow you to override the default width to go back to what it was before here:
![image](https://github.com/user-attachments/assets/d5415679-1b33-44b3-b4a3-a483960faf94)
For example:
![image](https://github.com/user-attachments/assets/193a2205-f431-42a7-b3d7-defb70592921)
Other wikis are similar. For example, the Arch Wiki:
![image](https://github.com/user-attachments/assets/0f6b57ef-2bca-4806-8856-116c47f70b79)

**Comment by Nutomic** [comment]
Interesting, English wikipedia has a max width but German wikipedia doesnt. Anyway I will leave it like this for now, and do a bigger redesign in the future.
